mix.exs

```diff
@@ -27,6 +28,13 @@ defmodule Craft.MixProject do
     ]
   end
 
+  defp aliases do
+    [
+      "compile.nif":
+        "cmd cc -fPIC -shared -dynamiclib -undefined dynamic_lookup -I /opt/homebrew/Cellar/erlang/27.3.3/lib/erlang/usr/include -o priv/clock_wrapper.so c_src/clock_wrapper.c"
```

Comment:
you've got a static path in here, that's not portable. it'll only work for your specific combination of package manager and erlang version